.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,137 @@
+        path: ~/.npm
+
+    - id: sdkman
+      if: steps.grails-sdk.outputs.cache-hit != 'true'
+      name: download grails SDK (unless cached)
+      uses: sdkman/sdkman-action@master
+      with:
+        candidate: grails
+        version: 1.3.9
+
+    - if: steps.grails-sdk.outputs.cache-hit != 'true'
+      name: unzip grails SDK (unless cached)
+      run: unzip -o ${{ steps.sdkman.outputs.file }} -d ~/
+
+    - name: npm install
+      run: npm install
+
+    - name: npm rebuild node-sass
+      run: npm rebuild node-sass
+
+    - name: npm run bundle
+      run: npm run bundle
+
+    - name: configure OpenBoxes for database access and testing
+      run: |
+        mkdir -p ~/.grails
+        echo "dataSource.password=${{ env.OPENBOXES_PASSWORD }}" > ~/.grails/openboxes-config.properties
+        echo "dataSource.url=jdbc:mysql://localhost:3306/${{ env.DB_DATABASE }}?allowPublicKeyRetrieval=true&useSSL=false" >> ~/.grails/openboxes-config.properties
+        echo "dataSource.username=${{ env.OPENBOXES_USER }}" >> ~/.grails/openboxes-config.properties
+        echo "openboxes.fixtures.enabled=true" >> ~/.grails/openboxes-config.properties
+        echo "openboxes.jobs.dataCleaningJob.enabled=false" >> ~/.grails/openboxes-config.properties
+        echo "openboxes.jobs.refreshDemandDataJob.enabled=false" >> ~/.grails/openboxes-config.properties
+        echo "openboxes.jobs.refreshInventorySnapshotJob.enabled=false" >> ~/.grails/openboxes-config.properties
+        #
+        # FIXME Strangely, enabled = false doesn't disable refreshProductAvailabilityJob,
+        # FIXME but setting delayInMilliseconds to a high value has the same practical effect.
+        #
+        echo "openboxes.jobs.refreshProductAvailabilityJob.enabled=false" >> ~/.grails/openboxes-config.properties
+        echo "openboxes.jobs.refreshProductAvailabilityJob.delayInMilliseconds=3600000" >> ~/.grails/openboxes-config.properties
+        echo "openboxes.refreshAnalyticsDataOnStartup.enabled=false" >> ~/.grails/openboxes-config.properties
+
+      #
+      # Liquibase migrations from scratch are very, very slow. For performance,
+      # this step pre-loads mysqldump output from a clean install of a recent
+      # OpenBoxes release. Technically, this step is optional in the sense that
+      # later steps would automatically run all the necessary migrations if it
+      # were removed. But those migrations would certainly take hours, and almost
+      # certainly fail with out-of-memory errors.
+      #
+    - name: preload empty database (v0.8.18-hotfix3)
+      run: |
+        gunzip -c ${{ github.workspace }}/test/data/preload.tgz | tar xvf -
+        mysql -p${{ env.DB_PASSWORD }} -u${{ env.DB_USER }} "${{ env.DB_DATABASE}}" < preload_schema.sql
+        mysql -p${{ env.DB_PASSWORD }} -u${{ env.DB_USER }} "${{ env.DB_DATABASE}}" < preload_data.sql
+
+      # if this step fails midway through, the next will pick up where it left off
```

Comment:
This guards against `No such property: mergedConfig for class: org.codehaus.groovy.grails.commons.DefaultGrailsApplication`, which I ran into when developing this action, but which I can't reproduce anymore. Maddeningly intermittent Docker build failures are, well, maddening... I think `continue-on-error: true` can't possibly hurt in this context and with any luck will prevent build gotchas down the line

Reply:
Feel free to make suggestions if you have any thoughts about how to fix that issue (or any of these issues (https://github.com/openboxes/openboxes-docker/issues)

Reply:
You bet. If it comes up again in Grails 3 I'll look at it a bit more closely.